### ISSUE TITLE
1.6.2: sphinx warnings

### ISSUE THREAD

**Issue by kloczek** [sissaschool/xmlschema#248]
```console
[tkloczko@barrel xmlschema-1.6.2]$ /usr/bin/python3 setup.py build_sphinx -b man
/usr/lib64/python3.8/distutils/dist.py:274: UserWarning: Unknown distribution option: 'license_file'
  warnings.warn(msg)
running build_sphinx
Running Sphinx v4.0.2
making output directory... done
WARNING: html_static_path entry '_static' does not exist
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 9 added, 0 changed, 0 removed
reading sources... [100%] usage
WARNING: autodoc: failed to import exception 'XMLSchemaException' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLResourceError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaNamespaceError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaValidatorError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaNotBuiltError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaParseError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaModelError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaModelDepthError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaValidationError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaDecodeError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaEncodeError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaChildrenValidationError' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaIncludeWarning' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaImportWarning' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import exception 'XMLSchemaTypeTableWarning' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'validate' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'is_valid' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'iter_errors' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'to_dict' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'to_json' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'from_json' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XMLSchema' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XMLSchemaBase' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XsdGlobals' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'ElementData' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XMLSchemaConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'UnorderedConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'ParkerConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'BadgerFishConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'AbderaConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'JsonMLConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'ColumnarConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'DataElement' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'DataElementConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'DataBindingConverter' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'fetch_resource' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'fetch_schema' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'fetch_schema_locations' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import function 'normalize_url' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XMLResource' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XmlDocument' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'ElementPathMixin' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.ValidationMixin' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.ParticleMixin' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XsdComponent' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XsdType' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XsdElement' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'XsdAttribute' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Element' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Attribute' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11ComplexType' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdComplexType' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdSimpleType' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAtomicBuiltin' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdList' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Union' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdUnion' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11AtomicRestriction' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAtomicRestriction' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAttributeGroup' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Group' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdGroup' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11AnyElement' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAnyElement' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11AnyAttribute' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAnyAttribute' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdOpenContent' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdDefaultOpenContent' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdIdentity' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdSelector' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdFieldSelector' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Unique' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdUnique' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Key' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdKey' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.Xsd11Keyref' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdKeyref' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdWhiteSpaceFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdLengthFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdMinLengthFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdMaxLengthFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdMinInclusiveFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdMinExclusiveFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdMaxInclusiveFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdMaxExclusiveFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdTotalDigitsFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdFractionDigitsFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdExplicitTimezoneFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAssertionFacet' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdEnumerationFacets' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdPatternFacets' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAssert' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAlternative' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdNotation' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'validators.XsdAnnotation' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'extras.codegen.AbstractGenerator' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'extras.codegen.PythonGenerator' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
WARNING: autodoc: failed to import class 'extras.wsdl.Wsdl11Document' from module 'xmlschema'; the following exception was raised:
cannot import name 'translate_pattern' from 'elementpath' (/usr/lib/python3.8/site-packages/elementpath/__init__.py)
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... xmlschema.1 { intro usage features converters components testing extras api } done
build succeeded, 100 warnings.
```

**Comment by brunato** [comment]
Hi, 
this is probably the same dependency problem as in #247.

**Comment by kloczek** [comment]
After fixeing dependencies still I see 2 warnings
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
/usr/lib64/python3.8/distutils/dist.py:274: UserWarning: Unknown distribution option: 'license_file'
  warnings.warn(msg)
running build_sphinx
Running Sphinx v4.0.2
making output directory... done
WARNING: html_static_path entry '_static' does not exist
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 9 added, 0 changed, 0 removed
reading sources... [100%] usage
/home/tkloczko/rpmbuild/BUILD/xmlschema-1.6.2/xmlschema/validators/schema.py:docstring of xmlschema.XMLSchemaBase.url:1: WARNING: duplicate object description of xmlschema.XMLSchemaBase.url, other instance in api, use :noindex: for one of them
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-xmlschema.3 { intro usage features converters components testing extras api } done
build succeeded, 2 warnings.
```

**Comment by brunato** [comment]
It's a duplicate `    .. autoattribute:: url` in *api.rst*.
The other warnings should disappear after updating dependencies with `$ pip install --upgrade -r requirements-dev.txt`.

**Comment by brunato** [comment]
Warning fixed with release v1.6.3.

**Comment by kloczek** [comment]
Tested 1.8.0. 
LGTM. Thank you :)
